content/techniques/http-module.md

```diff
@@ -147,3 +147,29 @@ export class CatsService {
   }
 }
 ```
+
+#### Full Example
+
+Since the return of `HttpService` is an Observable, we can use `rxjs` `firstValueFrom` or `lastValueFrom` to retrieve the data of the request for us.
```

Comment:
```suggestion
Since the return of `HttpService` is an Observable, we can use `rxjs` `firstValueFrom` or `lastValueFrom` to retrieve the data of the request in the form of a promise.
```